### PR TITLE
support tf::SinOp and tf::CosOp

### PR DESCRIPTION
**Pull request by wyzero**

### CODE REVIEW

**Review by qiuxiafei — APPROVED** [review]

**Comment by Orion34-lanbo** [comment]
Does tf community provide conversion from `tf.sin` to `mhlo::sine`, so we do not need to add converter in our lower tf pass?

File: tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc [hunk 1]
```diff
@@ -2595,10 +2596,10 @@ std::vector<string> GetDiscSupportedOps() {
     "Less",
     "LessEqual",
     "Log",
+    "LogSoftmax",
```
Comment (Yancey1989) [inline tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc:2599]: do we need a log_softmax ut ?

**Comment by wyzero** [comment]
> Does tf community provide conversion from `tf.sin` to `mhlo::sine`, so we do not need to add converter in our lower tf pass?

yes. I also added some e2e UTs to ensure this behaviour.

File: tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc (hunk 1 above)
Comment (wyzero) [reply tao/tao_bridge/passes/tao_mark_for_compilation_pass.cc:2599]: `LogSoftmax` is not introduced by this PR actually. I just sorted the ops in alphabetic order.

**Pull request merged**